spec/acceptance/nodesets/default.yml

```diff
@@ -32,16 +30,13 @@ HOSTS:
 
 
   el8:
-    roles:
-      - default
-      - master
-      - client
     platform: el-8-x86_64
     # box: centos/8 does not work.  When switching from
-    # selinux disabled to enabled it does nto reboot. It
+    # selinux disabled to enabled it does not reboot. It
     # hangs up relabeling.  Possibly because /boot is not
     # a seperate partition???
-    box: generic/centos8
+    #box: generic/centos8
```

Comment:
should this comment be removed?  Reverted to centos/8 which implies the problem is fixed?